geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ConvertCommand.scala

```diff
@@ -0,0 +1,110 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools
+
+import java.io.{FileInputStream, _}
+import java.util.zip.Deflater
+
+import com.beust.jcommander.{ParameterException, Parameters}
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.commons.io.IOUtils
+import org.geotools.feature.DefaultFeatureCollection
+import org.geotools.filter.text.ecql.ECQL
+import org.locationtech.geomesa.convert.SimpleFeatureConverters
+import org.locationtech.geomesa.tools.ConvertParameters.ConvertParameters
+import org.locationtech.geomesa.tools.export.{OptionalBinExportParams, _}
+import org.locationtech.geomesa.tools.export.formats._
+import org.locationtech.geomesa.tools.utils.DataFormats._
+import org.locationtech.geomesa.tools.utils.{CLArgResolver, DataFormats}
+import org.locationtech.geomesa.utils.text.TextTools.getPlural
+import org.opengis.filter.Filter
+
+class ConvertCommand extends Command with LazyLogging {
+
+  override val name = "convert"
+  override val params = new ConvertParameters
+
+  override def execute() = {
+    import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType.RichSimpleFeatureType
+    import scala.collection.JavaConversions._
+
+    val fmtParam = Option(params.format).flatMap(f => DataFormats.values.find(_.toString.equalsIgnoreCase(f)))
+    lazy val fmtFile = params.files.flatMap(DataFormats.fromFileName(_).right.toOption).headOption
+    val fmt = fmtParam.orElse(fmtFile).orNull
```

Comment:
future refactoring - we could move this logic into the new converter/input files params trait you added - think it's probably duplicated in several places now